include/rdma/fi_eq.h

```diff
@@ -73,9 +71,12 @@ struct fid_wait {
 	struct fi_ops_wait	*ops;
 };
 
+struct pthread_mutex_t;
+struct pthread_cond_t;
```

Comment:
My concern has never been about whether this change _worked_, but whether it is understandable and _appropriate_, especially in libfabric's public headers.  Thanks for reverting it.